### PR TITLE
Chore add necessary programming tools

### PR DESCRIPTION
**Pull request by maciejSamerdak**
I went through scripts located in **golemfactory/concent** as advised and after gaining some surface-level understanding of what is going on there, I wrote my scripts in the same fashion. I'm not sure however whether this is the desired outcome of this whole chore. Thus I'm posting this PR already for general feedback on possible misinterpretation or inappropriate configuration.

Also, I already included configuration for applications, which are planned to be added following week.

### CODE REVIEW

File: coverage-config [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[run]
+omit=
+	manage.py
+	*/__init__.py
+    	*/admin.py
+    	*/apps.py
+    	*/setup.py
+    	*/urls.py
+    	*/wsgi.py
+    	*/management/*
```
Comment (kbeker) [inline coverage-config]: It doesn't doesn't exist. Delete it

File: coverage-config [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[run]
+omit=
+	manage.py
+	*/__init__.py
+    	*/admin.py
+    	*/apps.py
+    	*/setup.py
```
Comment (kbeker) [inline coverage-config]: It doesn't exists. Delete it

File: full-check.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+printf "=============================== LINT ===============================\n"
+./lint.sh
+printf "\n"
+
+printf "========================= MYPY STATIC TYPE CHECKER =================\n"
+mypy --config-file=mypy.ini time_monkey/
+mypy --config-file=mypy.ini users/
+mypy --config-file=mypy.ini employees/
+mypy --config-file=mypy.ini managers/
```
Comment (kbeker) [inline full-check.sh]: None of them exists now. Please delete them

File: pylintrc [hunk 4]
```diff
@@ -0,0 +1,558 @@
...
+
+# Number of spaces of indent required inside a hanging  or continued line.
+indent-after-paren=4
+
+# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1
+# tab).
+indent-string='    '
+
+# Maximum number of characters on a single line.
+max-line-length=100
```
Comment (kbeker) [inline pylintrc]: Change it to 120

File: run-test-coverage.sh [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+rm .coverage
+cd ..
+
+cd managers
+pytest 					\
+	--cov-report term-missing 	\
+	--cov-config ../coverage-config \
+	--cov=.
+rm .coverage
+cd ..
```
Comment (kbeker) [inline run-test-coverage.sh]: All of these still does not exist. Delete them. Add comment that it needs to be updated when any of these modules will be added.

File: pylintrc [hunk 6]
```diff
@@ -0,0 +1,558 @@
...
+        next-method-defined,
+        dict-items-not-iterating,
+        dict-keys-not-iterating,
+        dict-values-not-iterating,
+        deprecated-operator-function,
+        deprecated-urllib-function,
+        xreadlines-attribute,
+        deprecated-sys-function,
+        exception-escape,
+        comprehension-escape
```
Comment (kbeker) [inline pylintrc]: Please add these ignores also:

```
    apply-builtin,
    missing-docstring,
    invalid-name,
    bad-whitespace,
    multiple-statements,
    protected-access,
    bad-continuation,
    redefined-variable-type,
    locally-disabled,
    locally-enabled,
    similarities,
    design,
    elif,
    logging,
    no-else-return,
    len-as-condition,
```

Check if it won't be duplicated.

File: coverage-config (hunk 1 above)
Comment (maciejSamerdak) [reply coverage-config]: Done

File: coverage-config (hunk 2 above)
Comment (maciejSamerdak) [reply coverage-config]: Done

File: full-check.sh (hunk 3 above)
Comment (maciejSamerdak) [reply full-check.sh]: Done

File: pylintrc (hunk 4 above)
Comment (maciejSamerdak) [reply pylintrc]: Done

File: run-test-coverage.sh (hunk 5 above)
Comment (maciejSamerdak) [reply run-test-coverage.sh]: Done

File: pylintrc (hunk 6 above)
Comment (maciejSamerdak) [reply pylintrc]: Done

File: pylintrc [hunk 7]
```diff
@@ -139,7 +139,23 @@ disable=print-statement,
...
+    	bad-continuation,
+    	redefined-variable-type,
+    	locally-disabled,
+    	locally-enabled,
+    	similarities,
+    	design,
+    	elif,
+    	logging,
+    	no-else-return,
+    	len-as-condition,
```
Comment (kbeker) [inline pylintrc]: Please fix indentation

Comment (maciejSamerdak) [reply pylintrc]: Done

File: lint.sh [hunk 8]
```diff
@@ -0,0 +1,6 @@
+printf "[PYLINT: concent_api, middleman_protocol, signing_service]\n"
+# Find all subdirectories of our python apps and use xargs to pass them as arguments to pylint
+
+find time_monkey/ users/ employees/ managers/  -maxdepth 1 -mindepth 1 -type d \
```
Comment (kbeker) [inline lint.sh]: Please delete directories which does not exists.

File: lint.sh [hunk 9]
```diff
@@ -0,0 +1,6 @@
+printf "[PYLINT: concent_api, middleman_protocol, signing_service]\n"
```
Comment (kbeker) [inline lint.sh]: Change names for current project

**Pull request merged**